app/src/main/java/org/gdg/frisbee/android/cache/ModelCache.java

```diff
@@ -150,7 +150,8 @@ public Object get(String url, boolean checkExpiration) {
         }
 
         if (result != null) {
-            return result.getValue();
+            //noinspection unchecked
+            return (T) result.getValue();
```

Comment:
I'll try but I'm not sure about that. The suppressing is about casting from Object